Add Find-my-Friends Service (#265)
picklepete/pyicloud: README.rst

>>> api.iphone.lost_device(phone_number, message)

Find-my-Friends
===============

> The Find-my-Friends service is lets you locate the friends you have you are following using the FindMe app. Once you have successfully authenticated, you can request the friends you are following and their location.

Reviewer: ```suggestion
The Find-my-Friends service lets you locate the friends you are following using the FindMe app.
```

Too complicated + repetitions

Friends
*******

> Returns the Find-my-Friends object.

Reviewer: ```suggestion
Returns the Find-my-Friends service.
```

>>> fmf = api.friends

> The fmf friends contact, location and following data is structured with the following dictionaries:

Reviewer: ```suggestion
The `contacts`, `following` and `locations` data is structured with the following dictionaries:
```

Contacts
********
Returns a dictionary of all contacts for the account.

>>> contacts = fmf.contacts

> Following Contacts

Reviewer: ```suggestion
Following contacts
```
No upper case in the middle of a sentence/title :relieved:

******************
Returns a dictionary of all of the contacts that are being followed. They have accepted a following invitation request from the account.

>>> following = fmf.following

Return the ID number and email address of the contact being followed. 

>>> contact_id = following['id']
>>> contact_email = following['invitationAcceptedHandles']

> 

Reviewer: Can you add the`fullName` ?

> Location of Followed Friends

Reviewer: ```suggestion
Location of followed friends
```
